simphony/testing/utils.py

```diff
@@ -80,6 +80,18 @@ def create_particles(n=10, restrict=None):
     return particle_list
 
 
+def create_particles_with_id(n=10, restrict=None):
```

Comment:
Since I think we will have to add them to the new `testing/test/test_utils.py` file that is not part of the master branch yet, can we add them after we merge #181 ?